lib/internal/Magento/Framework/Escaper.php

```diff
@@ -303,6 +303,11 @@ public function encodeUrlParam($string)
      */
     public function escapeJs($string)
     {
+        if (!is_string($string) && !is_array($string)) {
```

Comment:
In fact `escapeJs` should receive string and I believe that we will have real strict types here.
So I think we could use only string check here

Reply:
Hello @Den4ik 
Thank you for your proposition.
I think it would be better keep this checking here.